tests/codecheck/src_files.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 FAUCETHOME=`dirname $0`"/../.."
-for i in clib faucet tests ; do find $FAUCETHOME/$i/ -type f -name [a-z]*.py ; done | xargs realpath | sort
+for i in clib faucet tests ; do find $FAUCETHOME/$i/ -type f -name '[a-z]*.py' ; done | xargs realpath | sort
```

Comment:
This isn't strictly necessary for this patch, but it seems like a good idea. I may open a separate PR for it.